input/metadata/default_metadata.json

```
@@ -2,140 +2,140 @@
   "files": {
     "config": {
       "title": "Config Data",
-      "description": "The configuration file for the simulation.",
+      "description": "Configuration file for general simulation parameters, e.g. start and end dates.",
```

```suggestion
      "description": "Configuration file for general simulation parameters.",
```
The config file contains some pretty important stuff besides start and end dates, so I think not being specific about what it contains would be better. This pushes a user towards actually looking in it to see what it is doing and how it may be effecting their simulation, instead of just assuming that it is the start and end dates and that they don't need to worry about it.

---

Point well taken!